EIPS/eip-4950.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: ERC-721 extension with two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721
+---
+
+## Abstract
+This EIP standardizes an interface for creating entangled pairs of tokens. An entangled pair of tokens give his owners the right to operate with this common wallet.
+
+## Motivation
+The motivation is to provide an easy way to share a wallet through NFTs. So that the act of buying an NFT (in a marketplace) gives the buyer the privilege to have access to a given wallet. This wallet could have budget in many tokens, or even be the propietary of other NFTs.
+
+An use case is to keep contact between an artist and an buyer of its NFTs. If an artist T has created a digital piece of art P with an NFT, then T creates 2 entangled tokens A and B so that he keeps A and transfer B to P. By construction of entangled tokens, only one transfer is possible for them, thus the artist proofs he’s been the creator of P by sending a transaction to A that is visible from B. Otherwise, the owner of P might check the authenticity of the artist by sending a transaction to B so that the artist might proof by showing the outcome out of A.
+
+An extension of this first use case is when one user U and its creation in the form of an NFT T with wallet then the user U creates and transfers an entangled token A to T, and U keeps the entangled token B in the U’s wallet.  
+
+These applications of entangled tokens are envisaged to be useful for 
+1.	NFT authorship / art creation
+2.	Distribution of royalties by the creator.
+3.	Authenticity of a work of art: creation limited to the author (e.g. only 1000 copies if there are 1000 1000 entangled tokens in that NFT).
+4.	Usowners

```

Comment:
I am not familiar with this term.

Reply:
It means "users are owners of the NFT", so that an user consuming an NFT, becomes also owner (partially) of it

Reply:
Please include that definition in the EIP itself.

Reply:
Done